src/styles/stars.css

```diff
@@ -19,9 +13,6 @@
   margin-right: 4px;
 }
 
-.stars-1,
-.stars-2,
-.stars-3,
-.stars-4 {
+.stars--1 .stars__star:nth-child(-n+4) {
   background: url(../images/star-active.svg) no-repeat center;
 }
```

Comment:
Add the possibility to set different number of stars by changing the modifier. 

If I add modifier `stars--4` I'll see 4 active stars(3 for stars--3 and so on).

![Screenshot 2022-07-18 at 07 32 30](https://user-images.githubusercontent.com/17098736/179445739-33aed1f5-fbcb-4bab-817d-5bb20dc2ac79.png)